docs/specifications/tests/Basic-TP/basic02.md

```diff
@@ -1,45 +1,76 @@
-## BASIC02: The domain must have at least one working name server
+# BASIC02: The domain must have at least one working name server
 
-### Test case identifier
+## Test case identifier
 **BASIC02** The domain must have at least one working name server
 
-### Objective
+## Objective
 
 In order for the domain to work, it must have at least one name server that
 can answer queries about the domain. 
 
-### Inputs
+## Inputs
+
+* The label of the domain name to be tested ("child zone").
+* If undelegated test, the list of name servers ("undelegated NS").
+* If undelegated test, the IP addresses of the [in-bailiwick] undelegated 
+  NS, if any ("undelegated glue IP").
+* If undelegated test, the IP addresses of the [out-of-bailiwick]
+  undelegated NS, if any ("undelegated non-glue IP").
+
+## Ordered description of steps to be taken to execute the test case
+
+1. Retrieve the NS records for _the child zone_ (the delegation) using
+   [Method 2]. If the test is an undelegated test, use _undelegated
+   NS_.
+2. If the NS set is empty (no delegation), this test case fails
```

Comment:
Maybe use the same language here as in step 6? I.e. "If no NS records could be retrieved".